firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -582,6 +582,15 @@ void startLoadingConfigsFromDisk() {
     fetchedConfigsCache.get();
   }
 
+  /**
+   * Execute a runnable in Remote Config's background thread pool.
+   *
+   * @hide
+   */
+  public void schedule(Runnable runnable) {
```

Comment:
Have you tried making this method private so that we can remove the `@hide`?
I assume it works since the KTX function is under the same package name. But if it doesn't we can keep it as is.

Reply:
Yeah, I was hoping that, too but no luck. I think the `.ktx` sub-package means it needs to be public here.